### PR TITLE
PORTALS-1607

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/lib/style/base/_core.scss [hunk 1]
```diff
@@ -1223,3 +1216,74 @@ hr ~ .SRC-wrapper {
...
+    }
+  }
+
+  // do not apply special styling to dropdown links or anchors in tabs
+  .dropdown, .nav-tabs, .dropdown-menu {
+    a:not(.btn) {
+      @include ignoreLink;
+    }
+  }
+}
```
Comment (emmanmills) [inline src/lib/style/base/_core.scss:1289]: Have you checked if the new link style will render correctly for the top menu in mobile view (< 768px)? Thanks.

Screen Shot 2021-07-16 at 2 00 51 PM

Comment (jay-hodgson) [reply src/lib/style/base/_core.scss:1289]: Menu looks ok to me.  Thanks for bringing this up though, the style change could certainly have unintended side effects in buried UI like this.

Comment (jay-hodgson) [reply src/lib/style/base/_core.scss:1289]: Screen Shot 2021-07-16 at 2 17 19 PM

**Review by emmanmills — APPROVED** [review]
Thanks, looks good to me!

**Pull request merged**